### PR TITLE
RealtimeClient added

### PR DESCRIPTION
**Pull request by juancarlospaco**
## What kind of change does this PR introduce?

- Add `RealtimeClient` to have correct abstractions.

### CODE REVIEW

File: realtime/__init__.py [hunk 1]
```diff
@@ -1,6 +1,7 @@
 __version__ = "2.0.0"
 
 from realtime.channel import CallbackListener, Channel
+from realtime.client import RealTimeClient
```
Comment (J0) [inline realtime/__init__.py:4]: consider `realtimeClient` to mirror the swift lib https://github.com/supabase-community/realtime-swift

**Review by J0 — APPROVED** [review]

File: realtime/__init__.py (hunk 1 above)
Comment (grdsdev) [reply realtime/__init__.py:4]: note that that repo you linked is archived, swift lib is a monorepo and realtime is on https://github.com/supabase/supabase-swift/tree/main/Sources/Realtime

**Review by grdsdev — CHANGES_REQUESTED** [review]
Don't think we need this @juancarlospaco

Better just rename Socket to RealtimeClient.

**Comment by juancarlospaco** [comment]
Superseded.

**Pull request closed without merging**